docs/OPTIONS-LIST.md

```diff
@@ -284,9 +285,11 @@ For the allowed families arguments, you can either use the value:
 comma-separated values (without space) will allow every value inputted,
 every other value will be rejected
 - for numbers +/- text families (5.5,6.5,7.5 (komis), 9,13,19
-(boardsizes)), it is possible to use as well the "range"
+ (boardsizes)), it is possible to use as well the "range"
 operator `:` to navigate one by one from min to max (ex:
-`5.5:7.5` is `5.5,6.5,7.5` and `13:17` is `13,14,15,16,17`).
+ `5.5:7.5` is `5.5,6.5,7.5` and `13:17` is `13,14,15,16,17`),
+ as well as the "increment" operator (ex: `13:19:2` is `13,15,17,19`,
+see [notes A-](/docs/NOTES.md#a) for details.
```

Comment:
Do we need this dash after the A?

Reply:
no indeed, there was a dash because the linked used to be in the `/docs/NOTES.md/a-`, because the category title was `## A:`, but now since we removed the `:` we dont need the `a-` anymore, so we can just remove as well the `A-` 's dash